modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -201,6 +201,7 @@ ContentDashboardAdminManagementToolbarDisplayContext contentDashboardAdminManage
 
 							<liferay-ui:search-container-column-text>
 								<clay:dropdown-actions
+									cssClass="component-action"
```

Comment:
Now we have here the same problem we had with `dropdownCssClass="dropdown-action"`. This is something present in every actions dropdown so we shouldn't need to specify it in every usage.

Reply:
Also added it as part of the proposed solution.